src/main/java/htsjdk/samtools/cram/encoding/reader/DataReaderFactory.java

```diff
@@ -1,228 +0,0 @@
-/**
```

Comment:
`DataSeriesReader` now handles most of what this Factory was used for.  Some exceptions inline below.